### DISCUSSION TITLE
Kartenbild herunterladen & Allgemeine Tilecaches mit zufällig leeren Kacheln

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#27] [General]
z.B. 
https://gis.ktn.gv.at/webgisviewer/atlas-mobile/map/Basiskarten/0rientierung%20u.%20Kataster

Kartenbild herunterladen, Auflösung mittel/hoch: Das Ergebnis zeigt teilweise leere Orthofotokacheln. Wird "Kartenbild herunterladen" mit den gleichen Optionen erneut ausgeführt, sind andere Kacheln betroffen. Ist hier das Zeitfenster für WebGIS etwas zu eng definiert? Woran könnte es liegen?

![image](https://user-images.githubusercontent.com/79247887/213377278-701575da-137d-42fa-9d0f-dd20a0dbcc10.png)

**Comment by cmairamhof** [comment]
Im Ausdruck passt die Darstellung

**Comment by gis-eni** [comment]
Ist seither noch nirgends aufgetreten und kann ich nicht nachvollziehen. Sollte eigentlich gleich funktionieren wie der Ausdruck.
Sollten wir vielleicht weiter beobachten.
